lib/ContextHandler.js

```diff
@@ -35,6 +35,25 @@ module.exports = class ContextHandler {
       this.client = contextClient;
   }
 
+  /**
+   * @description This method gets a specified context into tenant layer
+   * @param {string} contextName The desirable context name
+   * @param {string} tenant The tenant who's the client belongs
+   * @returns a promise that:
+   * on success:
+   * gives an array which:
+   *   the first position is the context id, the user must store it to use in the
+   *     saveContext method;
+   *   the second position is an object with context content, the user can modify
+   *     it as your wish
+   * WARNING: you must call the saveContext method after use the context
+   * on failure:
+   *   gives a string the describes the error
+   */
+  getTenantContext(contextName, tenant) {
```

Comment:
Why the contextName first? It's seems more intuitive to be the tenant parameter.